### PR TITLE
Fix/floating devices

### PR DESCRIPTION
**Pull request by julsemaan**
# Description

Fix floating devices in active/active clusters
Make Juniper switch module use Net::SSH2 for floating devices
# Impacts

Floating devices
# Delete branch after merge

YES
## Enhancements
- Switched floating devices SSH commands to Net::SSH2 in Juniper EX2200 module
## Bug Fixes
- Fixed floating devices in active/active clusters

### CODE REVIEW

File: lib/pf/Switch/Juniper/EX2200.pm [hunk 1]
```diff
@@ -197,143 +197,118 @@ sub wiredeauthTechniques {
...
+    my $ssh = $self->_connectSSH();
+
+    return unless($ssh);
+
+    my $port = $ifIndex;
+
+    my $command_mac_limit = "set ethernet-switching-options secure-access-port interface $port mac-limit 16383";
+    my $command_disconnect_flap = "delete protocols dot1x authenticator interface $port mac-radius flap-on-disconnect";
 
     if ($@) {
```
Comment (dwlfrth) [inline lib/pf/Switch/Juniper/EX2200.pm]: Remove me

File: lib/pf/api.pm [hunk 2]
```diff
@@ -1427,6 +1427,70 @@ sub chi_cache_clear : Public {
...
+Call enableMABFloating in the local queue
+
+=cut
+
+sub enableMABFloatingInQueue : Public {
+    my ( $class, %postdata ) = @_;
+    my $logger = pf::log::get_logger;
+
+    my $client = pf::api::queue->new(queue => 'general');
+    $client->notify( '_enableMABFloating', %postdata );
```
Comment (dwlfrth) [inline lib/pf/api.pm]: Is the "_" on purpose ?

File: lib/pf/api.pm [hunk 3]
```diff
@@ -1427,6 +1427,70 @@ sub chi_cache_clear : Public {
...
+Call disableMABFloating in the local queue
+
+=cut
+
+sub disableMABFloatingInQueue : Public {
+    my ( $class, %postdata ) = @_;
+    my $logger = pf::log::get_logger;
+
+    my $client = pf::api::queue->new(queue => 'general');
+    $client->notify( '_disableMABFloating', %postdata );
```
Comment (dwlfrth) [inline lib/pf/api.pm]: Is the "_" on purpose ?

**Comment by julsemaan** [comment]
Fixed issues

**Pull request merged**